packages/mgt-components/src/components/mgt-person/strings.ts

```diff
@@ -26,6 +26,5 @@ const activityMap = {
 export const strings = {
   ...activityMap,
   photoFor: 'Photo for',
-  emailAddress: 'Email address',
-  initials: 'Initials'
```

Comment:
I think this is used for contacts without an image. So a11y _should_ announce "Initials MA"?

Reply:
That might have been the intent at one stage, but it is not used in the code, I searched in the code as I was trying to write a test for that string and found it unused.
I think that it might have been removed when we were working through how the narrator announces stuff with the a11y folks.